### ISSUE TITLE
Benchmarking no longer works after rust-lang/rust#68708

### ISSUE THREAD

**Issue by ssomers** [rust-lang/rust#68924]
#68708 suspended the ability to benchmark liballoc (and probably others, but not everything). `x.py bench` (with or without  --stage 0) now lists oodles of errors starting with:
```
Compiling getrandom v0.1.14
error: duplicate lang item in crate `core` (which `rustc_std_workspace_core` depends on): `bool`.
  |
  = note: first defined in crate `core` (which `getrandom` depends on)
```
It's still possible though to build and test benchmarks with say `x.py test --stage 0 --no-doc`.

**Comment by Mark-Simulacrum** [comment]
Hm, it could be that some of those Lang items need to be cfg(not(test)) or so.

**Comment by ssomers** [comment]
Issue appears resolved by #71439
